notify_send.py

```diff
@@ -160,6 +160,10 @@
         '/usr/share/icons/hicolor/32x32/apps/weechat.png',
         'Path to an icon to be shown in notifications.'
     ),
+    'desktop-entry': (
```

Comment:
Could you please also mention the new option in the _Options_ section in `README.md`?